src/main/java/com/softserve/kickscootertrip/service/TripService.java

```diff
@@ -0,0 +1,87 @@
+package com.softserve.kickscootertrip.service;
+
+import com.softserve.kickscootertrip.model.dto.TripDto;
+import com.softserve.kickscootertrip.model.dto.UserDto;
+import com.softserve.kickscootertrip.model.entity.TripEntity;
+import com.softserve.kickscootertrip.repository.TripRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class TripService {
+
+    private static final double RADIUS_EARTH = 6371008.67;
+    private static final String SALES = "TripForSales";
+    private static final String PAYMENT = "TripForPayment";
+    private final TripRepository tripRepository;
+    private ConversionService convert;
```

Comment:
change name from verb to noun.

Reply:
note for everyone: don't shorten variable names. The best way is to put some sense in the variable name to document code. But in this case (naming of bean names) just you the one provided by IDE -> conversionService.